x-pack/plugins/security_solution/public/management/pages/policy/view/policy_hooks.ts

```diff
@@ -88,3 +89,16 @@ export const useIsPolicySettingsBarVisible = () => {
     window.location.pathname.includes('/settings')
   );
 };
+
+/**
+ * Indicates if user is granted Write access to Policy Management. This method differs from what
+ * `useUserPrivileges().endpointPrivileges.canWritePolicyManagement` in that it also checks if
+ * user has `canAccessFleet`. This is to ensure that the Policy Form remains accessible when
+ * when displayed inside of Fleet pages if the user does not have privileges to security solution
+ * policy management.
+ */
+export const useCanWritePolicyManagementOrHasFleetAccess = (): boolean => {
+  const { canWritePolicyManagement, canAccessFleet } = useUserPrivileges().endpointPrivileges;
+
+  return canWritePolicyManagement || canAccessFleet;
```

Comment:
This should be `(canWritePolicyManagement && !canAccessFleet) || (!canWritePolicyManagement && canAccessFleet)`. No?

Reply:
🤔 I don't think so. If a user has both Endpoint policy management and they also have fleet access, then this would turn false, which is not correct - right?

If a user has `canAccessFleet` but no endpoint `canWritePolicyManagment`, although this will evaluate to true, the user will still not be able to access the endpoint page since we block access based on the `canReadPolicyManagement`.

Maybe I should rename this hook so that it is clearer that it applies to the form fields only?

Reply:
yeah, I'll refactor it and rename it before taking it out of draft